php_phongo.c

```diff
@@ -233,18 +233,32 @@ static void phongo_cursor_init(zval *return_value, mongoc_cursor_t *cursor, mong
 	intern = Z_CURSOR_OBJ_P(return_value);
 	intern->cursor = cursor;
 	intern->server_id = mongoc_cursor_get_hint(cursor);
-	intern->client = client;
+	intern->client = Z_MANAGER_OBJ_P(manager)->client;
+
+#if PHP_VERSION_ID >= 70000
+	ZVAL_COPY(&intern->manager, manager);
```

Comment:
Are you sure this just doesn't create a shallow copy of the ZVAL - and hence doesn't really affect the ref counting of the manager structs stored in it?